src/app/(main)/client.tsx

```diff
@@ -7,7 +7,7 @@ import { useEffect } from "react";
 
 export default function Client() {
     const projectsQuery = useQuery<paginatedResponse<Project>>({
-        queryKey: ["projects"],
+        queryKey: ["app:projects"],
```

Comment:
```suggestion
        queryKey: ["projects", "app"],
```
Recommended convention by react query